synfig-studio/src/gui/docks/dock_toolbox.cpp

```diff
@@ -231,6 +234,25 @@ Dock_Toolbox::add_state(const Smach::state_base *state)
 
 	refresh();
 }
+void Dock_Toolbox::new_layer(){
```

Comment:
1. Please rename the method. It seems it will create a new layer as soon as it is called.
2. Add a blank line separating a method implementation from another
3. Respect our (not well defined) code style for method implementation:
   - Return type in a line
   - Method name and parameters in the next line
   - Open curly bracket `{` in the next one.